### ISSUE TITLE
Add limited pause/trust  options

### ISSUE THREAD

**Issue by mathuaerknedam** [ghostery/ghostery-extension#24]
### Description
The pause button pauses Ghostery, across all pages. It would be nice to have the time-limited pause options for only one site/domain, or even better for only one browser tab. 

### Expected Behavior 
I expect to be able to pause Ghostery in order to diagnose a problem, complete a purchase, etc on a webpage, without disabling it everywhere.

### Actual Behavior 

I pause Ghostery to complete a specific task in a specific browser tab, but I'm exposed to tracking on any browser page that loads/reloads. An alternative would be to use the Trust button, but then I lose the ability to time-limit the change, and risk forgetting to undo the Trust.

### Steps to Reproduce

1. Navigate to Website A that doesn't work when blocking all trackers.
2. Unblock individual trackers in an attempt to figure out which one is needed.
3. Frustratedly pause Ghostery so that you can complete your task.
4. Get interrupted and go do something else on Website B, while exposed by the fact that Ghostery is paused EVERYWHERE.
7. Go back to Website A and complete your task

### Versions

+ Mac OS
+ Chrome / Firefox / Safari (Safari also lacks time-limited pause)

**Comment by christophertino** [comment]
This happens to me, too. I'll forget to resume until later on when I notice the Ghostery icon is grey.  I suppose the argument against tab-specific pause is that it overlaps with the Trust Site (white-list) feature. It might be interesting to consider a time-limited white-list. 

@tillhouse

**Comment by dmckenna24** [comment]
I agree, I think that this will be great for people who are forgetful and don't want to be whitelisted for an extended period of time.
